deflate.c

```diff
@@ -51,6 +51,27 @@
 
 #include "deflate.h"
 
+#ifdef __ARM_FEATURE_CRC32
+#  define ARM_CRC32_INTRINSIC
+#  include <arm_acle.h>
+#endif
+
+#ifndef FASTEST
+#if defined(ARM_CRC32_INTRINSIC)
+#  define USE_CRC32_INTRINSIC
+#  define EXTRA_DIST 1
+/* EXTRA_DIST is the difference of MIN_MATCH */
+#  undef MIN_MATCH
+#  define MIN_MATCH 4
+#endif
+#endif
```

Comment:
Redefining MIN_MATCH should be done in zutil.h as it is also used in trees.c. As far as I understand, this means defining ARM_CRC32_INTRINSIC should also be moved to a header file.